### PR TITLE
feat: replace bigint and BigUint64Array

### PR DESCRIPTION
**Pull request by HaydenOrz**
To solve #39

### CODE REVIEW

File: src/misc/BitSet.ts [hunk 1]
```diff
@@ -201,27 +189,26 @@ export class BitSet implements Iterable<number>{
...
     };
 
-    private bitCount(v: bigint): number { // a.k.a. hamming weight
-        v = v - ((v >> 1n) & 0x5555555555555555n);
-        v = (v & 0x3333333333333333n) + ((v >> 2n) & 0x3333333333333333n);
-        v = (v + (v >> 4n)) & 0x0f0f0f0f0f0f0f0fn;
-        v = v + (v >> 8n);
-        v = v + (v >> 16n);
-        v = v + (v >> 32n);
+    private bitCount(v: number): number {
```
Comment (mike-lischke) [inline src/misc/BitSet.ts:204]: Please keep the comment.

**Review by mike-lischke — CHANGES_REQUESTED** [review]

**Review by mike-lischke — APPROVED** [review]

**Comment by mike-lischke** [comment]
@HaydenOrz seems there's a problem with signing one of your commits.

**Comment by HaydenOrz** [comment]
> @HaydenOrz seems there's a problem with signing one of your commits.

Now the signature of the commit has passed. I didn't know much about commit signing before, so it took me a while.

**Comment by mike-lischke** [comment]
Great, thanks.

**Pull request merged**